src/dao/dynamo/index.ts

```diff
@@ -1,184 +1,184 @@
-import { isObject } from '../../object';
-import dbClient from 'aws-dynamodb-factory-js';
-import { error } from '../../error';
+import { isObject } from '../../object'
+import dbClient from 'aws-dynamodb-factory-js'
+import { error } from '../../error'
 
 interface Params {
-  TableName: any,
-  Key?: any,
-  Limit?: number,
-  ExclusiveStartKey?: object,
-  ExpressionAttributeNames?: object,
+  TableName: any
+  Key?: any
+  Limit?: number
+  ExclusiveStartKey?: object
+  ExpressionAttributeNames?: object
 }
 
 interface ResponseBatch {
-  UnprocessedItems: object,
+  UnprocessedItems: object
 }
 
 export class DynamoDBGenericDao {
-  static async get({
+  static async get ({
     params,
-    fields = [],
-  }: { params: Params, fields?: Array<string> }): Promise<object> {
+    fields = []
+  }: { params: Params, fields?: string[] }): Promise<object> {
     const { Item } = await dbClient.doc().get({
       ...params,
-      ...this.mountProjectionExpression({ fields, options: params }),
-    }).promise();
+      ...this.mountProjectionExpression({ fields, options: params })
+    }).promise()
 
-    return Item;
+    return Item
   }
 
-  static async query({
-    params, fields = [], _items = [], stopOnLimit = false,
-  }: { params: Params, fields?: Array<string>, _items?: Array<object>, stopOnLimit?: boolean }): Promise<Array<object>> {
+  static async query ({
+    params, fields = [], _items = [], stopOnLimit = false
+  }: { params: Params, fields?: string[], _items?: object[], stopOnLimit?: boolean }): Promise<object[]> {
     const { Items, LastEvaluatedKey } = await dbClient.doc().query({
       ...params,
-      ...this.mountProjectionExpression({ fields, options: params }),
-    }).promise();
+      ...this.mountProjectionExpression({ fields, options: params })
+    }).promise()
 
     const newItemsList = (Array.isArray(Items) && Items.length > 0)
       ? _items.concat(Items)
-      : _items;
+      : _items
 
-    if (stopOnLimit && params.Limit && newItemsList.length >= params.Limit) return newItemsList;
+    if (stopOnLimit && params.Limit && newItemsList.length >= params.Limit) return newItemsList
```

Comment:
Unexpected nullable number value in conditional. Please handle the nullish/zero/NaN cases explicitly.

```suggestion
    if (stopOnLimit && params.Limit !== undefined && newItemsList.length >= params.Limit) return newItemsList
```